hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconAsPassiveScm.java

```diff
@@ -81,7 +81,7 @@ public void shutdown() {
     }
   }
 
-  @Test(timeout = 120000)
+  @Test
```

Comment:
We should have some timeout value, it's ok to increase it if needed.

Reply:
Timeout is implicitly enforced through LambdaTestUtils.